package.json

```diff
@@ -85,11 +86,11 @@
     "react-scripts-start": "PORT=3100 BROWSER=none react-scripts start",
     "serve": "NODE_ENV=production ts-node -O '{\"module\": \"commonjs\", \"typeRoots\": [\"./src/@types\"]}' scripts/server.tsx",
     "serve-dev": "NODE_ENV=development ts-node-dev -O '{\"module\": \"commonjs\", \"typeRoots\": [\"./src/@types\"]}' -- scripts/server.tsx",
-    "stage": "cat .env.common-local .env.stage > .env.local && yarn start-dev",
+    "stage": "yarn build-local-stage-config && yarn stmux [ 'yarn test' : 'tsc -w' : 'yarn react-scripts-start' : 'yarn serve-dev' ]",
```

Comment:
Everything including and after `yarn stmux` looks like it could be extracted into a shared script, maybe named `dev-stmux` or something.

Reply:
thank you